packages/nextjs-mf/src/loaders/helpers.ts

```diff
@@ -1,8 +1,11 @@
 import type { RuleSetRuleUnion, Loader } from '@module-federation/utilities';
 
 /**
- * Inject a loader into the current module rule.
- * This function mutates `rule` argument!
+ * This function injects a loader into the current module rule.
+ * Note: This function mutates the `rule` argument.
+ *
+ * @param {RuleSetRuleUnion} rule - The current module rule.
+ * @param {Loader} [loader={}] - The loader to be injected.
```

Comment:
The function 'injectRuleLoader' can be improved by checking if 'rule' is '...' at the beginning of the function and returning early if true. This can improve readability and maintainability.